### PR TITLE
 integ test against local real network + using kotlin in test

### PR DESCRIPTION
**Pull request by yosriz**
* launch local network using stellar-ops repo
* change urls to locals
* make kotlin available for tests only
* use multi dex in instrumentation tests (reached 65k method limit)
* migrate KinAccountIntegrationTests and FakeKinIssuer to kotlin

### CODE REVIEW

File: kin-core/src/androidTest/kotlin/kin/core/FakeKinIssuer.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    @Throws(IOException::class)
+    private fun createAndFundWithLumens(accountId: String) {
+        val client = OkHttpClient()
+        val request = okhttp3.Request.Builder()
+                .url(URL_CREATE_ACCOUNT + accountId).build()
+        val response = client.newCall(request).execute()
+        val succeeded = response?.body() != null && response.code() == 200
+        response?.let {
+            if (!succeeded) {
```
Comment (mrsegev) [inline kin-core/src/androidTest/kotlin/kin/core/FakeKinIssuer.kt]: Can be merged with line 35

**Review by mrsegev — APPROVED** [review]

File: kin-core/build.gradle [hunk 2]
```diff
@@ -51,8 +58,12 @@ dependencies {
     androidTestImplementation testingDependencies.androidTestRunner
     androidTestImplementation testingDependencies.androidTestRules
     androidTestImplementation testingDependencies.hamcrest
+    androidTestImplementation testingDependencies.multidex
+    androidTestImplementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
```
Comment (OrenZak) [inline kin-core/build.gradle]: why not from the dependencies?

File: kin-core/src/androidTest/java/kin/core/IntegConsts.java [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 final class IntegConsts {
```
Comment (OrenZak) [inline kin-core/src/androidTest/java/kin/core/IntegConsts.java:4]: can object with consts ?

File: dependencies.gradle [hunk 4]
```diff
@@ -11,6 +11,8 @@ ext {
     hamcrestVersion = '1.3'
     gsonVersion = '2.4'
     jacocoVersion = '0.8.0'
+    multiDexVersion = '1.0.3'
+    rxjavaVersion = '2.1.7'
```
Comment (OrenZak) [inline dependencies.gradle]: wtf ? rxjava and i don't know about it?

File: kin-core/build.gradle (hunk 2 above)
Comment (yosriz) [reply kin-core/build.gradle]: done

File: kin-core/src/androidTest/java/kin/core/IntegConsts.java (hunk 3 above)
Comment (yosriz) [reply kin-core/src/androidTest/java/kin/core/IntegConsts.java:4]: there's Java class that use that, don't want to touch for now

File: dependencies.gradle (hunk 4 above)
Comment (yosriz) [reply dependencies.gradle]: removed... I thought I need it but eventually not

**Pull request merged**